jina/docker/hubapi.py

```diff
@@ -117,6 +117,46 @@ def _list(logger, image_name: str = None, image_kind: str = None,
         return manifests
 
 
+def _docker_auth(logger) -> Optional[Dict[str, str]]:
+    """ Use Hub api to get docker creds
+
+    :return: a dict of specifying username and password
+    """
+    with resource_stream('jina', '/'.join(('resources', 'hubapi.yml'))) as fp:
+        hubapi_yml = JAML.load(fp)
+        hubapi_url = hubapi_yml['hubapi']['url'] + hubapi_yml['hubapi']['docker_auth']
+
+    access_token = _fetch_access_token(logger)
+    if not access_token:
+        logger.error(f'user has not logged in. please login using command: {colored("jina hub login", attrs=["bold"])}')
+        return
+
+    headers = {
+        'Accept': 'application/json',
+        'authorizationToken': access_token
+    }
+
+    try:
+        import requests
+        response = requests.get(url=f'{hubapi_url}',
+                                 headers=headers)
+        if response.status_code == requests.codes.ok:
+            import json
+            import base64
```

Comment:
Please move `json` & `base64` imports to the top of the file. 
Since `requests` is a 3rd party package, we can wrap that around `ImportExtensions` like [this](https://github.com/jina-ai/jina/blob/master/jina/docker/hubio.py#L55)